sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_analysis_client.py

```diff
@@ -137,13 +136,12 @@ def begin_analyze_document_from_url(self, model, document_url, **kwargs):
             Use this to specify the custom model ID or prebuilt model ID. Prebuilt model IDs to use are:
             "prebuilt-receipt", "prebuilt-invoice", "prebuilt-idDocument", "prebuilt-businessCard",
             "prebuilt-document", "prebuilt-layout".
-        :param str document_url: The URL of the document to analyze. The input must be a valid, encoded URL
-            of one of the supported formats: JPEG, PNG, PDF, TIFF, or BMP.
+        :param str document_url: The URL of the document to analyze. The input must be a valid, publicly
+            accessible URL of one of the supported formats: JPEG, PNG, PDF, TIFF, or BMP.
```

Comment:
Paul felt it was unclear what encoded url meant, which is why I removed it instead of explaining more. I thought it was kind of obvious, but if users are running into issues with this then I can add it back in.

Reply:
Done, I'll add an issue to try to improve this doc string a bit more.

Reply:
Here's the issue to discuss whether the description needs to be improved: #20953